etna/records/templatetags/records_tags.py

```diff
@@ -46,6 +46,31 @@ def as_label(record_field_name: str) -> str:
 
 
 @register.simple_tag
-def level_name(level_code: int) -> str:
+def level_name(level_code: int, is_tna: bool) -> str:
     """returns level as a human readable string"""
-    return LevelKeys["LEVEL_" + str(level_code)].value
+    if is_tna:
+        return LevelKeys["LEVEL_" + str(level_code)].value
+    else:
+        return NonTNALevelKeys["LEVEL_" + str(level_code)].value
+
+
+@register.simple_tag
+def breadcrumb_items(hierarchy: list, is_tna: bool, current_item: Record) -> list:
+    """Returns breadcrumb items depending on position in hierarchy
+    Update tna_breadcrumb_levels or oa_breadcrumb_levels to change the levels displayed
+    """
+    items = []
+    tna_breadcrumb_levels = [1, 2, 3]
```

Comment:
Can make use of constants, instead of hardcoding, eg LevelKeys.LEVEL_1

Reply:
Hi @TNA-Allan, this is so that levels can be specifically chosen to be displayed in the "breadcrumb"/non-detailed view (at the top of the hierarchy)

I'm not sure using the constants will work here as that would return a string - doing this allows us to pick specific levels (e.g. level 1, 2 and 3 as we have here) 😃

Reply:
@jamesbiggs , you're right it does return strings :)